src/emulator/functionsEmulator.ts

```diff
@@ -1316,8 +1330,24 @@ export class FunctionsEmulator implements EmulatorInstance {
           )} --save-dev" in your functions directory`,
         );
       } else {
+        let port: number;
+        if (typeof this.args.debugPort === "number") {
+          port = this.args.debugPort;
+        } else {
+          // Start the search at port 9229 because that is the default node
+          // inspector port and Chrome et. al. will discover the process without
+          // additional configuration. Other dynamic ports will need to be added
+          // manually to the inspector.
+          port = await portfinder.getPortPromise({ port: 9229 });
+          this.logger.logLabeled(
+            "SUCCESS",
+            "functions",
+            `Using debug port ${port} for functions codebase ${backend.codebase}`,
```

Comment:
For ports other than 9299, maybe remind users that they will have to add the port to inspector manually in this message - ie:
```
Using debug port 9300 for functions codebase ${backend.codebase}. You may need to add manually add this port to your inspector.
```